source/core/lethe_grid_tools.cc

```diff
@@ -1373,14 +1372,14 @@ template <int dim>
 double
 LetheGridTools::find_point_triangle_distance(
   const std::vector<Point<dim>> &triangle,
-  const Point<dim> &             pos)
+  const Point<dim>              &point)
 {
-  auto &p_0 = triangle[0];
-  auto &p_1 = triangle[1];
-  auto &p_2 = triangle[2];
+  auto &point_0 = triangle[0];
+  auto &point_1 = triangle[1];
+  auto &point_2 = triangle[2];
```

Comment:
Here I meant for the auto -> Point<dim>, but not important

Reply:
Ohh no I can change that :)